src/index.html

```diff
@@ -4,10 +4,33 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg"
+      alt="product" class="card__photo">
+      <a href="#" class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+      <div class="card__code">Product code: 195434</div>
```

Comment:
use semantic tags for text content
```suggestion
      <p class="card__code">Product code: 195434</p>
```